api/spec/requests/spree/api/v2/storefront/account/addresses_spec.rb

```diff
@@ -49,10 +49,10 @@
     end
 
     context 'when address can not be deleted' do
+      let!(:order) { create(:completed_order_with_totals, bill_address: address) }
```

Comment:
I was getting `address can not be destroyed error`, so I made order undeletable in other way.